packages/cdktf/lib/tokens/private/stack-trace.ts

```diff
@@ -13,5 +13,5 @@ export function captureStackTrace(below?: Function): string[] {
   if (!object.stack) {
     return [];
   }
-  return object.stack.split('\n').slice(1).map(s => s.replace(/^\s*at\s+/, ''));
+  return object.stack.split('\n').slice(1).map(s => s.replace(/^\s*at\s+/, '').replace(/\(.*packages/, '(packages').replace(/\(.*node_modules/, '(node_modules'));
```

Comment:
What's this about?

Reply:
It was an attempt to work around inconsistent traces, but really it is an incomplete hack that is at best unrelated and at worst obscures where errors occur. I'll take it out.

Reply:
As mentioned here, I think that's a bug https://github.com/hashicorp/terraform-cdk/pull/241#issuecomment-664402253 - Let me check